lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -533,48 +530,55 @@ abstract class DefaultTextEditingStrategy implements TextEditingStrategy {
   @visibleForTesting
   bool isEnabled = false;
 
-  html.HtmlElement domElement;
-  InputConfiguration _inputConfiguration;
-  EditingState _lastEditingState;
+  html.HtmlElement get domElement => _domElement!;
+  html.HtmlElement? _domElement;
+
+  /// Overrides the HTML element used for text editing.
+  void useDomElement(html.HtmlElement element) {
+    _domElement = element;
+  }
```

Comment:
question: previously we are using "super.domElement = domElement;" why we switched to this method?

Reply:
I'm not sure any more :) Reverted.